conf/nginx/st2.conf

```diff
@@ -40,6 +40,10 @@ server {
 
   add_header              Front-End-Https on;
   add_header              X-Content-Type-Options nosniff;
+  add_header              X-Frame-Options DENY always;
+  add_header              Strict-Transport-Security "max-age=3153600;includeSubDomains";
+  add_header              Cache-Control "no-store; no-cache; max-age=0; must-revalidate;";
```

Comment:
While the security-focused headers are all good, I'm wondering what would be the user impact for the Cache-Control changes here? @ashwini-orchestral what are the specific reasons behind hardening cache-control this way?

WDYT @StackStorm/maintainers @StackStorm/contributors? Any downsides, any behavior/UX changes you could think of?

Reply:
Yeah, +1, I don't see much benefit security wise from cache control headers, it would just slow things down if we need to re-fetch all the static resources on each page (re)load...

But others (iframe, stric transport security, etc.) are indeed a good idea :+1: 

On that note, I believe we need to sync up multiple places (if they don't contain those headers already) - Docker, installer, etc. Or do those places use config from this repo?

Reply:
Right, we're pulling nginx config from the `st2` repo (this file).

Reply:
@ashwini-orchestral can you please remove the `Cache-Control` headers then?